parse_info.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+#Open jsonfile
+open_f=open('data.json')
```

Comment:
this needs to be saving into the data folder not in the current directory

Reply:
So to be sure it would save to the jsondata file right?

Reply:
![image](https://user-images.githubusercontent.com/82005267/220185567-aa452c61-c8a9-48f6-b40a-be36b128cfea.png)
I'm guessing something like this?